org.eclipse.jface.text/.settings/.api_filters

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <component id="org.eclipse.jface.text" version="2">
+    <resource path="META-INF/MANIFEST.MF">
+        <filter comment="An API was effectively added, LinkedModeUI.doExit, but some other filter are necessary and make API tool miss this" id="926941240">
```

Comment:
> but some other filter are necessary

Not sure I understood this sentence
It is important **if** that missing API error wan an API tools problem, report it in PDE.